editor/scene_tree_editor.cpp

```diff
@@ -1206,11 +1206,10 @@ SceneTreeEditor::SceneTreeEditor(bool p_label, bool p_can_rename, bool p_can_ope
 	}
 
 	tree->connect("cell_selected", callable_mp(this, &SceneTreeEditor::_selected_changed));
-	tree->connect("item_edited", callable_mp(this, &SceneTreeEditor::_renamed), varray(), CONNECT_DEFERRED);
+	tree->connect("item_edited", callable_mp(this, &SceneTreeEditor::_renamed), varray());
```

Comment:
You can omit varray() parameter too if it's empty.

Reply:
Done. Seems to work just fine. 
As a side-note, should I have pushed this as a separate commit instead of squashing it?

Reply:
No, you need to squash it. The practice of non-squashing is only for an advanced contribution where it's important to differentiate between the commits (and even then it can be avoided).